networking_monitoring.prolific

```diff
@@ -97,6 +97,99 @@ L: security
 
 ---
 
+Inspect your c2c packets with tcpdump
+
+### What?
+Sometimes a user says "container to container networking is broken! AppA can't talk to AppB". After making sure that they have c2c policies set, the next thing you might do is use tcpdump to inspect the packets.
+
+Tcpdump is a CLI tool that allows you to inspect all of the traffic (not just TCP, despite the name) flowing through your container. This is a great tool to use to debug how far packets are making it, how long the packets are taking to reach their destination, and to look at where in the TCP handshake things are failing.
+
+In this story you are going to look at the packets being sent from Boots to Swiper. Then you'll watch the packets being sent in response.
+
+### How?
+**Setup**
+1. You should have two apps pushed named Boots and Swiper.
+1. There should be no c2c network policies (`cf network-policies` and `cf remove-network-policies --help`).
+
+**Curl swiper from boots**
+1. Get the overlay IPs of boots and swiper.
+ ```
+cf ssh boots -c "env | grep CF_INSTANCE_INTERNAL_IP"
+cf ssh swiper -c "env | grep CF_INSTANCE_INTERNAL_IP"
+ ```
+1. Ssh onto boots.
+ ```
+cf ssh boots
+ ```
+1. Continually try to curl swiper from boots
+ ```
+watch -n 15 curl SWIPER_OVERLAY_IP:8080
+ ```
+
+**Look at those packets**
+1. In another terminal, ssh onto the Diego Cell where boots is running and become root (see the help section if you don't know how to do this).
+1. Run `tcpdump`.
+    Ahhhhh too much information! ctrl+c! ctrl+c!  On a Diego Cell there are many packets being sent around, and tcpdump gives information about ALL OF THEM. You need to figure out a way to filter this overwhelming stream of information.
+1.  Filter by packets where the source IP is BOOTS_OVERLAY_IP and where the destination IP is SWIPER_OVERLAY_IP over any interface. (Interested in learning more about network interfaces? Contact Amelia Downs about CF Networking Onboarding week! Or google it.)
+ ```
+tcpdump -n src BOOTS_OVERLAY_IP and dst SWIPER_OVERLAY_IP -i any
+ ```
+ Hey! Those are your packets!
+
+ Record the packets you see here from one curl.
+ ```
+# PUT TCPDUMP OUTPUT HERE
+ ```
+
+If swiper was successfully responding, then you should also see packets being sent as a response in the opposite direction.
+
+1. See that no packets are being sent back from Swiper to Boots
+  ```
+tcpdump -n src SWIPER_OVERLAY_IP and dst BOOTS_OVERLAY_IP -i any
+ ```
+
+**Add c2c policy**
+1. Add c2c policy to allow traffic from Boots to Swiper (`cf add-network-policy --help`)
+1. Continually try to curl Swiper from Boots
+
+**Look at those packets**
```

Comment:
nit: Could also say "inspecting packets" or "inspect packets". Look at those packets is also good, just thinking of other wording that doesn't use "those" (which implies a specific set of packets)